sjsonnet/src/sjsonnet/Materializer.scala

```diff
@@ -25,7 +25,7 @@ abstract class Materializer {
       case obj: Val.Obj =>
         storePos(obj.pos)
         obj.triggerAllAsserts(obj)
-        val objVisitor = visitor.visitObject(obj.visibleKeyNames.length , -1)
+        val objVisitor = visitor.visitObject(obj.visibleKeyNames.length , jsonableKeys = true, -1)
```

Comment:
note: pay attention here `jsonableKeys = true` comes from lib update. I checked and I think true is a valid value